templates/privacy_policy.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+
+{% block title %}Privacy policy — {{ block.super }}{% endblock %}
+
+{% block meta_tags %}
+  <meta property="og:title" content="Privacy policy">
+  <meta property="og:description" content="">
+  <meta name="description" content="">
+  <meta property="og:url" content="{% url 'privacy-policy' %}">
+{% endblock meta_tags %}
+
+{% block breadcrumb %}
+  <ul class="breadcrumbs list-unstyled">
+    <li>
+      <a href="{% url 'home' %}">Home</a>
+    </li>
+    <li>
+      <a href="{% url 'privacy-policy' %}">Privacy policy</a>
+    </li>
+  </ul>
+{% endblock breadcrumb %}
+
+{% block content %}
+  <h2>Privacy policy</h2>
+  <p>The Saleor Demo store is operated by Mirumee Software sp. z o.o. sp. k. operating at the following address:</p>
+  <address>
+    Mirumee Software sp. z o.o. sp. k.<br>
+    ul. Tęczowa 7<br>
+    53-601, Wrocław<br>
+    POLAND
+  </address>
+  <h2>Analytics</h2>
+  <p>This website includes Google Analytics software that collects anonymized information about visitors to help us provide better services. <a href="https://www.google.com/policies/privacy/partners/">See how Google uses your data.</a></p>
+  <h2>Personal data</h2>
+  <p>This website allows address data consisting of given and email addresses, family names, shipping and billing addresses to be entered for the purpose of testing the checkout process.
```

Comment:
`consisting of given and email addresses` -> `consisting of given names and email addresses` ?

Reply:
My bad, it should say: "(...) of given and family names, email addresses, (...)"